rosa_planning/rosa-hcp-prereqs.adoc

```diff
@@ -0,0 +1,87 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
Preview link: https://88263--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs#rosa-sts-aws-prereqs

Reply:
OR maybe this module is not published yet? Am I in the wrong doc set. Is HCP a new doc set?

Reply:
The ROSA docs are being split into separate Classic and HCP docs; the preview link above is for the Classic docs (which do currently still contain some HCP things that are not yet split)